### ISSUE TITLE
Features for classifying an ontology - replace reference vs. application

### ISSUE THREAD

**Issue by ramonawalls** [OBOFoundry/OBOFoundry.github.io#400]
The simple classification of references versus application ontology is ambiguous and no longer very useful. At the operations committee on 3/7/17, we discussed having a type of checklist for classifying axes along different axes. Below is a first draft of said list. Feel free to chime in with other suggestions.

Purpose (not necessarily mutually exclusive):
-Intended to serve as a reference for a broad domain (e.g., animal anatomy, biomedical research) -Intended to serve as a reference for a specific domain (e.g., biobanks, plant phenology) 
-Intended for a specific application, but may be useful more broadly
-Intended for a specific applicaiton, probably not useful more broadly

Term requests:
-Accepts term requests.
-Accepts term requests but may funnel them to a more general ontology
-Accepts term requests but only funnels them to a more general ontology
-Does not accept term requests

Terms in own namespace:
-Yes
-No
-% terms in the ontology’s namespace

MIREOTs:
-Yes
-No
-%of terms imported from other ontologies

Integration: Does the ontology add OWL axioms that connect external terms?
-Yes
-No
-Number of added axioms connected to imported terms.

**Comment by cmungall** [comment]
This is great. What are the next actions?

 - Create new fields in the yaml for some of the self-described aspects in the upper part of the list
 - Write SPARQL for the lower part (derived)
 - Report somehow on the site

**Comment by nlharris** [comment]
Is this something that still should be done?

**Comment by nlharris** [comment]
What is the status of this? Is this still something we want?

**Comment by matentzn** [comment]
Holy!!! I didnt realise this issue was discussed as early is 2017! And nothing ever happened. We absolutely should address this.. See also #1225

**Comment by matentzn** [comment]
Oh maaaaan! No dictators out there that are willing to take the plunge and just decide that there are reference ontologies and "other" ontologies, and "other" ontologies terms have to be ignored by term browsers and curation tools? It would be so useful to have these categories in OBO...

**Comment by matentzn** [comment]
"Application ontologies" will mint IDs. They just need to be ignored by downstream tools!

And we still need a name for this.

**Comment by nataled** [comment]
The only input I can provide on this--with regard to a historical perspective--is that from the beginning of the current OBO governance structure (Operations, TWG, EWG, OWG), the EWG understood that the principles definitely applied to reference ontologies, but might not work for application ontologies, and that there might be tweaks (or even a few new ones) needed for the latter.

**Comment by nlharris** [comment]
I think this will be addressed by the new home page ontology tables, right?

**Comment by nlharris** [comment]
I believe this is now addressed by the new home page, right?

**Comment by nataled** [comment]
@nlharris I'd say not, as there's nothing about the new page that addresses the issue. If anything, it somewhat hides it. Besides, this goes beyond website considerations. Indeed, as noted above, the EWG long ago recognized that the differences will affect how ontologies are reviewed (automated or not).

**Comment by nlharris** [comment]
Ok.

**Comment by nlharris** [comment]
I still think this is now addressed by the new(ish) obofoundry.org page organization, but @nataled may still disagree.

**Comment by cthoyt** [comment]
I still don't think this is addressed

**Comment by nataled** [comment]
Yep, I still disagree. I'm not sure what it is about the new page organization that allows us to distinguish reference vs application.
